Parent/Parent/Course/CourseDetailsViewController.swift

```diff
@@ -79,14 +84,31 @@ class CourseDetailsViewController: HorizontalMenuViewController {
         viewControllers.append(summaryViewController)
     }
 
+    func configureFrontPage() {
+        let vc = CoreWebViewController()
+        vc.webView.loadHTMLString(frontPages.first?.body ?? "", baseURL: nil)
+        viewControllers.append(vc)
+    }
+
     func courseReady() {
-        if !courses.pending && readyToLayoutTabs, !didLayoutTabs, let course = courses.first {
+        if !courses.pending && !frontPages.pending && readyToLayoutTabs, !didLayoutTabs, let course = courses.first {
             didLayoutTabs = true
             configureGrades()
-            if let body = course.syllabusBody, !body.isEmpty {
-                configureSyllabus()
-                configureSummary()
+
+            switch course.defaultView {
+            case .syllabus:
+                if let body = course.syllabusBody, !body.isEmpty {
+                    configureSyllabus()
+                    configureSummary()
+                }
+            case .wiki:
+                if let page = frontPages.first, !page.body.isEmpty {
+                    configureFrontPage()
+                    configureSummary()
```

Comment:
We shouldn't show summary here. The summary tab and the syllabus tab go hand-in-hand so it should only be shown when the syllabus is shown.

Reply:
fixed